content/docs/quickstart/creating-an-event-against-an-asset/index.md

```diff
@@ -19,52 +19,177 @@ Asset Creation is the first Event. The more Events recorded against an Asset, th
 
-For example:
+In the attributes section you should also add the required RKVST attributes `arc_description` and `arc_display_type` to represent `Event Description` and `Event Type`.
 
+```yaml
+---
+steps:
+  - step:
+      action: EVENTS_CREATE
+      description: Record event against My First Container.
+      asset_label: assets/<asset-id> 
+    operation: Record
+    behaviour: RecordEvidence
+    event_attributes:
+      arc_description: Inspection Event
+      arc_display_type:  Inspection
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+
+3. You may enter both Event and Asset attributes.
+
+* `Event Attributes` - Attributes specific to an Event, i.e. which device recorded the Event
+* `Asset Attributes` - Attributes of the Asset that may change as a result of the Event, i.e. overall weight of a container
+
+
+{{< tabs name="add_event_attr" >}}
+{{{< tab name="UI" >}}
+Select the `Add Attribute` button on each tab to add your key-value pairs.
 {{< img src="EventAttributes.png" alt="Rectangle" caption="<em>Event Specific Attributes</em>" class="border-0" >}}
 
 {{< img src="EventAssetAttributes.png" alt="Rectangle" caption="<em>Event Asset Attributes</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="YAML" >}}
+
+Add your `event_attributes` and `asset_attributes` as key-value pairs.
+```yaml
+---
+steps:
+  - step:
+      action: EVENTS_CREATE
+      description: Record event against My First Container.
+      asset_label: assets/<asset-id>
+    operation: Record
+    behaviour: RecordEvidence
+    event_attributes:
+      arc_description: Inspection Event
+      arc_display_type:  Inspection
+      Cargo: Rare Metals
+    asset_attributes:
+      Weight: "1192kg"
+    confirm: true
+```
+{{< /tab >}}}
+{{< /tabs >}}
+
+
 
 Here we see someone noted the type of cargo loaded in the Event, and has also recorded the total weight of the cargo using a newly defined `Weight` attribute.
 
 Every Event has an automatically generated `timestamp_accepted` and `principal_accepted` attribute that records _when_ who performed what, as submitted to RKVST.
 
-There is an option to append `timestamp_declared` and `principal_declared` attributes on the Event. For example, if the Event happened offline or a third party reports it. This option helps to create a detailed record.
+There is an option to append `timestamp_declared` and `principal_declared` attributes on the Event. For example, if the Event happened offline or a third party reports it, creating a more detailed record.
 
-PDFs or images can be recorded with an Event in the same way as an Asset. 
+PDFs or images can be recorded with an Event in the same way as an Asset. This is useful for storing associated material for posterity. For example, each `Inspection` Event can store the PDF document of a specific standard for container inspection. This allows historical compliance checking of Events.
```

Comment:
@CamrynCarter this mostly looks good but it would be cool to see an example of how to do attachments here - do you mind adding this?

Reply:
Yep- attachments added to example